packages/cli/src/options/beaconNodeOptions/index.ts

```diff
@@ -5,6 +5,7 @@ import * as api from "./api.js";
 import * as chain from "./chain.js";
 import * as eth1 from "./eth1.js";
 import * as execution from "./execution.js";
+import * as builder from "./builder.js";
```

Comment:
Please sort alphabetically to ease checking all options are present

Reply:
done